spinnaker_components/common/input_filtering.h

```diff
@@ -96,6 +96,9 @@ static inline void _if_filter_input(if_filter_t *filter,
 /* Simulate one step of a filter and reset its accumulator if necessary */
 static inline void _if_filter_step(if_filter_t* filter)
 {
+  // Disable interrupts to avoid a race condition
+  uint32_t cpsr = spin1_int_disable();
+
```

Comment:
`spin1_fiq_disable` would be more precise. `spin1_int_disable` will also work though (in as much as it sets the F and I flags).